src/htmxl/token.py

```diff
@@ -37,14 +38,18 @@ def parse_str(cls, data: str):
 
     @classmethod
     def from_node(cls, node):
+
+        classes = node.attrs.get("class", [])
+
         if node.name == "[document]":
-            return cls(node=node, name="body")
+            return cls(node=node, name="body", classes=classes)
```

Comment:
i dont think we have to worry about the document having classes. you can just do this after/for the 2nd return